### ISSUE TITLE
Cannot login with the user "Login Failed" after installing using Calamares

### ISSUE THREAD

**Issue by shah-newaz** [pieroproietti/penguins-eggs-legacy#228]
Hi,

I've cloned my Manjaro installation. It used to work fine until the latest update to 9.4.4. (Last time I tried and it worked was in 9.3.x).

But now, if I have 2 users, it deletes one user completely upon installation.
Then I removed the secondary user and cloned with only one user. It works fine on the live USB. But when I install it on a laptop with Calamares, I cannot login to the user anymore.

**Comment by pieroproietti** [comment]
Hi shah-newaz

on manjaro I have this versions: you can check from here https://packages.manjaro.org/?query=Penguins

Unfortunately for same reasons, eggs-v9.4.7 don't go yet on community-

Anyway, You can clone the newest version and build it with `makepkg -si`. All you have to do is:
* `git clone https://gitlab.manjaro.org/packages/community/penguins-eggs`
* `cd penguins-eggs`
* `makepkg -si`

I'm advising @Ste74 too.

**Comment by shah-newaz** [comment]
Thanks a lot for your amazing work and quick reply! I'm going to try this and update this thread @pieroproietti

**Comment by Ste74** [comment]
Pushed in stable now .. Thank you Mark

**Comment by shah-newaz** [comment]
Sadly, the issue persists... With latest

**Comment by shah-newaz** [comment]
The live session from USB runs just fine. No issues. I can login and all. It's after I install the distro in HDD is when this "Login Failed" starts happening.

**Comment by shah-newaz** [comment]
Okay, at least I know the cause. Calamares/installation not copying the home folder at all during installation. So my /home is just empty.

**Comment by shah-newaz** [comment]
Investigating a little more - it seems to be ignoring the file exclude list.

**Comment by pieroproietti** [comment]
Hi shah-newaz

If you are not using `--clone` option, its perfecty normal who "calamares/installation not copying the home folder at all", your user will be a new user!

Anyway I'm just now trying to check manjaro, I will start with the last version: manjaro-xfce-22.1.0-230421-linux61.iso, to see that happen.

After installed it, just:
* `sudo pacman -Syu`
* `sudo pacman -S penguins-eggs`
* `sudo eggs calamares --install`
* `sudo eggs dad --default`
* `sudo eggs tools clean`
* `sudo reboot`
* `sudo eggs produce`

Now I'm installing it with user: piero

![just-instaling](https://user-images.githubusercontent.com/958613/236245750-9056d7dd-3975-430c-9734-f2a8b638eb91.png)

After installed:
![this-is-the-installed-system](https://user-images.githubusercontent.com/958613/236246505-3148f29d-513e-4d29-b43f-d891eac33949.png)

**Comment by pieroproietti** [comment]
Now I will test it with `--clone` option, to let you to save the user and use it in the live and the new installation.

Before I add some stuffs inside home, for example: `git clone https://github.com/pieroproietti/penguins-blog`

![git-clone-some-sfutffs](https://user-images.githubusercontent.com/958613/236247328-44385543-e2c0-4ff5-ac3c-4912f3b2457f.png)

Now I will create a new iso with --clone option:

`sudo eggs produce --clone`

![sudo-eggs-produce--cloneScreenshot](https://user-images.githubusercontent.com/958613/236247889-2ac21529-bc61-40ab-866e-30480b0914bd.png)

**Comment by pieroproietti** [comment]
This is the live after the boot...
![boot-live](https://user-images.githubusercontent.com/958613/236251921-1203b05e-cc6c-4517-8b42-9336b72017ac.png)

Now we will install with calamares, note: becouse it's  a clone will not request to configure users...

![installed](https://user-images.githubusercontent.com/958613/236252302-6b2d3470-b855-4227-bb14-e7243f18213c.png)


Now I will reboot and see the results on the new installed system...

![new-system](https://user-images.githubusercontent.com/958613/236252950-89f60eb3-5e7c-4b2b-8fb9-74b8dcbddc25.png)

All seem OK shah-newaz... 

This is all I did... eggs was installed from pacman and it's the last, all I did was done now 4 may 2023.

Have nice time, and happy hacking

**Comment by shah-newaz** [comment]
@pieroproietti Thank you so much!
I actually did it a lot of times. This is my favorite tool. It used to work before and I used --clone of course. Sorry I forgot to mention.

Now I think it has something to do with my overused distro (I keep creating new remasters out of this almost every week :D ).

I'll try a fresh installation. 

Thank you so much again! So happy someone realised the importance of such a tool for Linux. Now I have a thumb drive with all my important dev apps installed in it. Portable!
